moderation/migrations/0005_auto_20190412_0442.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.18 on 2019-04-12 09:42
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
```

Comment:
What is this migration for ? I dont see any changes in models file. Why do you need this migrations ?

Reply:
They are auto-generated by django... don't understand why. There are no changes in the `models.py`... Did you forget to commit them earlier?

Reply:
ok I see.